Skills.md

```diff
@@ -38,6 +39,24 @@ JSON
   ]
 }
 ```
+If the return_relevant_skills request parameter is set to true, the reponse will look like this:
+
+```
+{
+	"response": [{

```

Comment:
hmm.. this is the jsonlint format. Is there another json validator you use for formatting?